nodejs/graphql/handlers/subgraph.handler.js

```diff
@@ -0,0 +1,34 @@
+require('dotenv').config({path: '.env'});
+const urlHandler = require('../../handlers/url.handler');
+const axios = require('axios');
+
+const query = JSON.stringify({query: '{ transfers(first: 5) { id sender recipient amount }'});
+
+
+
+class SubgraphHandler {
+    getTransfers = async (isMainnet)=>{
+
+        try {
+            const response = await axios.post(`${urlHandler.getSubgraphURL(isMainnet)}/aba76682a6nb4K6bDDN6aa71N1c4NN8a/erc20-transfers-quickstart/`, {

```

Comment:
`ARKHIA_API_KEY` hardcoded maybe

Reply:
Yes lets definitely remove that end and get it from the .env

Reply:
@Paul-okpor i think u can put this in the class RestApiHandler since its a post ?

Reply:
I think its better here because i will be expanding the subgraph